tofhir-server/pom.xml

```diff
@@ -165,6 +165,15 @@
             <groupId>io.onfhir</groupId>
             <artifactId>tofhir-common</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.onfhir</groupId>
+            <artifactId>tofhir-rxnorm_2.13</artifactId>
+        </dependency>
+        <dependency>
+            <!-- TODO get rid of this dependency when the server supports injecting libraries in some other way -->
```

Comment:
This is OK for now, as you suggested; we might implement external library injection in a later release.

On the other hand, I suppose these are very specific functions for the requirements of the DT4H project. Even in this case, it might be an option to implement a general function library including these functions. What do you think? @suatgonul

Reply:
Do you mean having a library inside the toFHIR code base instead of adding it as a dependency?